README.md

```diff
@@ -1,2 +1,35 @@
+| region           | The AWS region.                                                                                                                                                                | `""`                         |
+| bucket           | The AWS bucket to watch.                                                                                                                                                       | `""`                         |
+| prefix           | The bucket prefix.                                                                                                                                                             | `""`                         |
+| clean-objects    | Whether to delete S3 objects after processing them.                                                                                                                            | `false`                      |
+| timeout          | The global timeout.                                                                                                                                                            | `""`                         |
+| influx-server    | The InfluxDB server address.                                                                                                                                                   | `""`                         |
+| influx-token     | The InfluxDB token.                                                                                                                                                            | `""`                         |
+| influx-org       | The InfluxDB org to write to.                                                                                                                                                  | `""`                         |
+| influx-bucket    | The InfluxDB bucket write to.                                                                                                                                                  | `""`                         |
+| timestamp-row    | The timestamp row in CSV.                                                                                                                                                      | `"timestamp"`                |
+| timestamp-layout | The layout to parse timestamp.                                                                                                                                                 | `"2006-01-02T15:04:05.000Z"` |
+| tag              | Tags to add to InfluxDB point. Could be of the form `--tag=foo` or `--tag='foo={tag:bar}'` to set a specific tag name.                                                         | `""`                         |
+| field            | Fields to add to InfluxDB point. Could be of the form `--field='foo={type:int,field:bar}'` if not specified, field is the same as row. type can be float, int, string or bool. | `""`                         |
```

Comment:
```suggestion
| field            | Fields to add to InfluxDB point. Should be of the form `--field='foo={type:int,field:bar}'`, if not specified, field is the same as row. Type can be float, int, string or bool. | `""`                         |
```

Reply:
I though you were going to use the --field='foo={type:int,row:bar} syntax, but with this one it is unclear what is the influxDB field name and what is the CSV row name. Maybe the comment should explain it ?